.github/workflows/pr-check-e2e.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   pr-checks:
     runs-on: ubuntu-latest
-    timeout-minutes: 10
+    timeout-minutes: 20
```

Comment:
Cypress tests cover almost every function, so it takes some time. However, the login process impacts heavily because keycloak sometimes responds in dozens of seconds. I have thought about a way of removing login step only for the testing, like disabling auth on backend, but I want to do it as a last resort.